src/bin/CMakeLists.txt

```diff
@@ -234,7 +234,12 @@ set_property(TARGET msquic PROPERTY FOLDER "${QUIC_FOLDER_PREFIX}libraries")
 
 if(WIN32)
     if(QUIC_UWP_BUILD)
-        target_link_libraries(msquic PUBLIC OneCoreUAP)
+        target_link_libraries(msquic PUBLIC OneCore)
```

Comment:
Why this change?

Reply:
I need to revert this actually. I'll do it next time I'm at a computer.